pytorch_lightning/metrics/classification/utils.py

```diff
@@ -0,0 +1,446 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
thinking about rename this module as "helpers"?
so we do not have too many utils in the metrics package :]

Reply:
Good idea yes